Emby.Server.Implementations/Library/InvalidAuthProvider.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using MediaBrowser.Controller.Authentication;
+using MediaBrowser.Controller.Entities;
+
+namespace Emby.Server.Implementations.Library
+{
+    public class InvalidAuthProvider : IAuthenticationProvider
+    {
+        public string Name => "InvalidOrMissingAuthenticationProvider";
+
+        public bool IsEnabled => true;
+
+        public Task<ProviderAuthenticationResult> Authenticate(string username, string password)
+        {
+            throw new Exception("User Account cannot login with this provider. The Normal provider for this user cannot be found");
```

Comment:
A more specific exception is preferred.

Reply:
Do we have one that fits better?

Reply:
Maybe `SecurityException`? I don't really know.

Reply:
SecurityException doesn't seem to exist. I think a generic Exception is good enough, it's logging why it failed anyways.

Reply:
Pretty sure it exists https://github.com/jellyfin/jellyfin/blob/master/MediaBrowser.Controller/Net/SecurityException.cs
;)

Reply:
Aah, I needed an import for that.